### PR TITLE
Awesome books: with ES6

### PR DESCRIPTION
**Pull request by VSeray**
Creating the Awesome books project with ES6:

- Use ES6 syntax.
- Divide code into modules and import them into the index.js file.
- Install Luxon and import as a module.

### CODE REVIEW

File: .gitignor [hunk 1]
```diff
@@ -0,0 +1 @@
+/node_modules
```
Comment (rustam-tolipov) [inline .gitignor]: - [x] GOOD JOB SO FAR 👏. You have successfully added your ` .gitignore` file but it is not working as expected. The reason why it's not working is that you have called it `.gitignor`. Please kindly fix this issue with adding e to end of your file name like this `.gitignore`

File: README.md [hunk 2]
```diff
@@ -1 +1,74 @@
-# NewAwesome-book
```
Comment (rustam-tolipov) [inline README.md:1]: - [x] There is another thing that needs to be fixed that you should have a folder called `modules`. I would recommend changing your `js` folder to `modules` to fix this issue.  You could also add more modules for smaller functions to make your code better 👋🍀

See this image
image

File: index.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
+import Book from './js/bclass.js';
+import { DateTime } from './node_modules/luxon/build/es6/luxon.js';
+
```
Comment (rustam-tolipov) [inline index.js:3]: - [OPTIONAL] WELL DONE 👍. You have successfully installed and imported a package called `luxon`. However, you forgot to use it after installing 😜. Please firstly see [Get the current date and time](https://moment.github.io/luxon/#/tour?id=get-the-current-date-and-time) and add the date to your footer like this image 🍀.

Example
image

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hi @VSeray , 

Good job so far!

### HIGHLIGHTS!

- No linters errors.💯
- Descriptive Readme.md. 💯


There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments above the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by elyor-doniyorov — APPROVED** [review]
Hi @VSeray, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**